theme/static/styles/layout.less

```diff
@@ -0,0 +1,259 @@
+/**
+* @stylesheet layout.less Layout
+* @parent Styles.baseline-elements 3
+
+* @description
+* Layout classes are used to structure the main parts of a page.
+**/
+
+
+/**
+* @styles header Header
+*
+* @demo demos/header.html
+**/
+header {
+	display: block;
+	background-color: @colorHeader;
+	color: @clear;
+	position: relative;
+}
+
+header {
+	.container {
+		position: relative;
+		height: @headerHeight;
+		text-align: left;
+	}
+}
+
+
+/**
+* @styles header-nav Header Nav
+*
+* @demo demos/header-nav.html
+**/
+header .nav {
+	list-style-type: none;
+	padding: 0;
+	margin: 0 0 0 15px;
+	display: inline-block;
+	position: relative;
+	top: -2px;
+	> li {
+		display: inline-block;
+		line-height: @headerHeight;
+		position: relative;
+		> a {
+			text-transform: uppercase;
+			text-decoration: none;
+			font-weight: bold;
+			font-size: 12px;
+			color: @clear;
+			.text-shadow;
+			padding: 5px 10px;
+			border: 1px solid transparent;
+		}
+		&:hover > a {
+			background-color: rgba(0, 0, 0, 0.1);
+			border: 1px solid rgba(0, 0, 0, 0.3);
+			border-radius: 2px;
+		}
+		&.active > a {
+			background-color: rgba(0, 0, 0, 0.35);
+			border: 1px solid rgba(0, 0, 0, 0.6);
+			border-radius: 2px;
```

Comment:
@MarcoBrunoBR this is not a gaiden file, it's the documentation template style. Disconsider the variable creation